examples/seq2seq/README.md

```diff
@@ -66,6 +66,15 @@ Summarization Tips:
 Datasets: Seq2SeqDataset will be used for all models besides MBart, for which MBartDataset will be used.**
 A new dataset is needed to support multilingual tasks.
 
+
+### Command Line Options
+
+To see all the possible command line options, run:
+
+```bash
+./finetune.sh --help
```

Comment:
why not `python finetune.py --help`? makes more sense to me.

Reply:
Either way works, but README.md uses the bash script in its usage examples:
https://github.com/huggingface/transformers/blob/master/examples/seq2seq/README.md#finetuning-training-params
Whatever you choose ideally let's make it consistent?